pyglmnet/pyglmnet.py

```diff
@@ -432,8 +432,27 @@ def fit_predict(self, X, y):
         """
         return self.fit(X, y).predict(X)
 
-    def score(self, y, yhat, ynull=None, method='deviance'):
-        """Score the model.
+    def score(self, X, y):
+        """Scoring function compatible with sklearn.
+        If multiple regularization parameters are fit,
+        an additional step is needed (e.g., take the mean)
+
+        Parameters
+        ----------
+        X: array, shape (n_samples, n_features)
+            Training data
+        y: array, shape (n_samples, [n_classes])
+            True value or label of training data
+
+        Returns
+        -------
+        Generate scores for each regularization parameter
+        """
+        yhats = self.predict(X)
+        return [self.general_score(y, yhat) for yhat in yhats]
+
+    def general_score(self, y, yhat, ynull=None, method='deviance'):
```

Comment:
I am adapting `score` to comply with `sklearn`, but keeping all functionality in `general_score`. Maybe I should keep everything in `score` but let it behave in different ways depending on the parameters provided - i miss real polymorphism